src/main/kotlin/no/nav/familie/ba/sak/kjerne/steg/VilkårsvurderingSteg.kt

```diff
@@ -34,6 +37,26 @@ class VilkårsvurderingSteg(
                 )
             )
         }
+        // midlertidig validering for helmanuell migrering
+        if (behandling.erHelmanuellMigrering()) {
+            val vilkårsvurdering = vilkårService.hentVilkårsvurdering(behandling.id)
+                ?: throw IllegalStateException("Fant ikke aktiv vilkårsvurdering for behandling ${behandling.id}")
+            val finnesDelBosted = vilkårsvurdering.personResultater.any {
+                it.vilkårResultater.filter { vilkårResultat -> vilkårResultat.vilkårType == Vilkår.BOR_MED_SØKER }
+                    .any { borMedSøker ->
+                        borMedSøker.utdypendeVilkårsvurderinger
+                            .contains(UtdypendeVilkårsvurdering.DELT_BOSTED)
+                    }
+            }
+            if (!finnesDelBosted) {
+                throw Feil(
```

Comment:
```suggestion
                throw FunksjonellFeil(
```
Så slipper vi alarmer :)